accounts/hw/src/trezor.rs

```diff
@@ -29,9 +29,9 @@ use ethkey::Signature;
 use hidapi;
 use libusb;
 use parking_lot::{Mutex, RwLock};
-use protobuf::{self, Message, ProtobufEnum};
+use prost::{DecodeError, Message};
```

Comment:
Maybe a silly suggestion, but there's `rlp::DecodeError` all over the codebase so maybe a rename here is in order, e.g. `ProstDecodeError`?